src/main/java/net/smelly/murdermystery/game/map/MMMapGenerator.java

```diff
@@ -1,9 +1,10 @@
 package net.smelly.murdermystery.game.map;
 
+import net.minecraft.registry.Registry;
```

Comment:
This import is unused.

Reply:
Fixed in 540609b1a89ca51445db637980a16d2e00484d79